components/docker-auth/cmd/authn/authentication.go

```diff
@@ -85,22 +86,37 @@ func validateToken(inToken string, cert []byte) (bool, error) {
 }
 
 func main() {
-	flag.Parse()
+	file, err := os.OpenFile(logFile, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	if err != nil {
+		log.Fatalf("Error opening file: %v", err)
+	}
+	defer func() {
+		err = file.Close()
+		if err != nil {
+			os.Exit(2)
+		}
+	}()
+	if err != nil {
+		log.Println("Error while closing the file :", err)
+		os.Exit(extension.ErrorExitCode)
```

Comment:
You can use log.Fatalln for print and exit with non-zero exit code.

Reply:
Fixed by #178